frontend/lib/intro_slides/location_request_button.dart

```diff
@@ -39,17 +51,21 @@ class _LocationRequestButtonState extends State<LocationRequestButton> {
     }
     switch (status) {
       case LocationStatus.denied:
-      case LocationStatus.deniedForever:
       case LocationStatus.notSupported:
         return ElevatedButton(
-          onPressed: _onLocationButtonClicked,
+          onPressed: () => _onLocationButtonClicked(settings),
           child: const Text("Ich möchte meinen Standort freigeben."),
         );
       case LocationStatus.whileInUse:
       case LocationStatus.always:
         return const ElevatedButton(
           onPressed: null,
-          child: Text("Standort ist bereits freigegeben."),
+          child: Text("Standort ist freigegeben."),
+        );
+      case LocationStatus.deniedForever:
+        return const ElevatedButton(
+          onPressed: null,
+          child: Text("Standortfreigabe ist deaktiviert."),
```

Comment:
This case is important and correct. For reference open Google Maps and always deny the permission. If you click the "Show my location on Map"-Button you have no options to directly reenable the feature.

Reply:
I disagree. If I deny it forever in Google Maps, at some point, the "enable permission" dialog doesn't show up, but in that case, a snackbar appears a link to the app's settings.
![image](https://user-images.githubusercontent.com/7025199/141776176-c3c5f193-8edb-417e-84a2-a06b2d8aaf4e.png)

Reply:
This is crazy :D In my google maps version there is no SETTINGS button. I change it like it is now in Google maps in #408

Reply:
But I'll leave the intro slides as is. No need to allow the user again to give permissions, after he denied them for 2 times.